traffic_ops/traffic_ops_golang/crconfig/handler.go

```diff
@@ -174,6 +174,12 @@ func snapshotHandler(w http.ResponseWriter, r *http.Request, deprecated bool) {
 			return
 		}
 		cdn = name
+	} else {
+		_, _, err := getCDNInfo(cdn, inf.Tx.Tx)
+		if err != nil {
```

Comment:
The `getCDNInfo` func call doesnt differentiate between database query error (lets say the db was down) or the cdn not existing.  Those would call for different errors being returns to the user.

I would suggest using https://github.com/apache/trafficcontrol/blob/master/traffic_ops/traffic_ops_golang/dbhelpers/db_helpers.go#L485 as it returns if it exists or not in a separate var